tools/wptrunner/wptrunner/executors/protocol.py

```diff
@@ -982,3 +981,17 @@ def update_virtual_pressure_source(self, source_type, sample):
     @abstractmethod
     def remove_virtual_pressure_source(self, source_type):
         pass
+
+class WebExtensionsProtocolPart(ProtocolPart):
```

Comment:
I think this should be renamed to BiDiWebExtensionProtocolPart and added next to other BiDi parts https://github.com/web-platform-tests/wpt/blob/master/tools/wptrunner/wptrunner/executors/protocol.py#L370 as it is specified in the WebDriver BiDi spec.